dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -60,7 +60,7 @@ def get_donor_teacher_banner_details
 
   # GET /api/v1/users/<user_id>/school_donor_name
   def get_school_donor_name
-    render json: @user.school_donor_name
+    render json: @user.school_donor_name.nil? ? 'null' : "\"#{@user.school_donor_name}\""
```

Comment:
nit: `inspect` would probably be easier here
```suggestion
    render json: @user.school_donor_name.nil? ? 'null' : @user.school_donor_name.inspect
```